backend/.env.dist

```diff
@@ -0,0 +1,6 @@
+FLASK_CONFIG=development
+SECRET_KEY=
```

Comment:
should these be empty?

Reply:
Yeah, this is just a template for what the actual `.env` file should look like